avocado/core/teststmpdir.py

```diff
@@ -0,0 +1,31 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+
+
+import os
+import shutil
+import tempfile
+
+
+class TestsTmpDir(object):
+
+    def __init__(self, varname='XXX_TESTS_COMMON_TMPDIR'):
```

Comment:
This was discussed in the mail thread. I started with `AVOCADO_` and I was told to not use it.